### PR TITLE
Issue 1347 use screensize

### PR DESCRIPTION
**Pull request by aashish24**

### CODE REVIEW

**Comment by durack1** [comment]
@aashish24 it seems that `garant` is having some problems, it's failed here and also on #1540..

**Comment by aashish24** [comment]
@doutriaux1 looks like one test is failing. Although I don't know how its related to your change.

**Comment by durack1** [comment]
@aashish24 as noted above it seems that [`garant` is having configuration problems](https://open.cdash.org/index.php?compare1=65&filtercount=2&field1=buildname%2Fstring&project=UV-CDAT&field2=buildstarttime%2Fdate&showfilters=0&limit=100&compare2=83&value1=122c4dd1-build1152-uvcdat-garant-linux-release&showfeed=0&display=project&value2=N), rather than failing the `ctest` suite..

**Comment by jbeezley** [comment]
Not sure the issue, but this is the cmake output:
```
CMake Error: Error in cmake code at
/home/buildbot/buildbot/uvcdat-garant-linux-release/source/testing/vcs/CMakeLists.txt:35:
Parse error.  Expected a command name, got quoted argument with text "${PYTHON_EXECUTABLE}".
[INFO] You did not download UVCMETRICS Test data not test will be run
-- Configuring incomplete, errors occurred!
```

**Comment by jbeezley** [comment]
There is definitely a syntax error at [that line](https://github.com/UV-CDAT/uvcdat/commit/122c4dd15b2c929e4e59135a5f047c78080a13ab#diff-149b54d8dd4ed881ef449a3e12ba525eR34).  I'm not sure why it isn't failing on the other machines.

**Comment by doutriaux1** [comment]
@jbeezley fixing now. Thx for spotting.

**Comment by aashish24** [comment]
@jbeezley I pushed a fix

**Comment by doutriaux1** [comment]
@aashish24 you beat me to it. Thx.

**Comment by aashish24** [comment]
:smile_cat:

**Comment by aashish24** [comment]
@doutriaux1  we are getting this: 

 self.__plot(arglist, keyargs)
  File "/home/buildbot/buildbot/uvcdat-garant-linux-release/build/install/lib/python2.7/site-packages/vcs/Canvas.py", line 3839, in __plot
    returned_kargs = self.backend.plot(*arglist, **keyargs)
  File "/home/buildbot/buildbot/uvcdat-garant-linux-release/build/install/lib/python2.7/site-packages/vcs/VTKPlots.py", line 445, in plot
    self.createRenWin(**kargs)
  File "/home/buildbot/buildbot/uvcdat-garant-linux-release/build/install/lib/python2.7/site-packages/vcs/VTKPlots.py", line 314, in createRenWin
    self.initialSize()
  File "/home/buildbot/buildbot/uvcdat-garant-linux-release/build/install/lib/python2.7/site-packages/vcs/VTKPlots.py", line 412, in initialSize
    bgX = int(screenSize[0]*.6)
TypeError: can't multiply sequence by non-int of type 'float'

**Comment by aashish24** [comment]
Looks like VTK still giving us a pointer.

**Comment by doutriaux1** [comment]
did that got reverted? I works in what uvcdat-master we had last week.

**Comment by doutriaux1** [comment]
@aashish24 garant seems to be hosed?

**Comment by doutriaux1** [comment]
@aashish24 why are travis builds suddenly failing because of wrong version of cmake?

**Comment by doutriaux1** [comment]
now goes with: https://github.com/UV-CDAT/uvcdat-testdata/pull/66

**Comment by doutriaux1** [comment]
@aashish24 @jbeezley no matter what `ctest -D Experimental` switch the branch back to `master` sooooo annoying....

**Comment by doutriaux1** [comment]
@aashish24 @jbeezley why does ctest resets the branch to master!

File: CMake/cdat_modules_extra/checkout_testdata.cmake [hunk 1]
```diff
@@ -55,194 +55,202 @@ if(NOT EXISTS "${TESTDATA_DIR}")
   endif()
 endif()
 
-# 2) Is TESTDATA_DIR a clone of TESTDATA_URL?
-execute_process(COMMAND
```
Comment (aashish24) [inline CMake/cdat_modules_extra/checkout_testdata.cmake]: What's changed?

**Comment by doutriaux1** [comment]
@aashish24 @jbeezley I got it! RH6 FULL is passing! Retriggering it on all machines now.

**Comment by aashish24** [comment]
what was the issue?

**Comment by doutriaux1** [comment]
xtra cmake args from commit were not parsed correctly by bot slaves

**Comment by doutriaux1** [comment]
I updated the slaves to send the command all the time, this latest commit doesn't send any extra args to slaves so we will see if the system works properly now.

File: CMake/cdat_modules_extra/checkout_testdata.cmake (hunk 1 above)
Comment (doutriaux1) [inline CMake/cdat_modules_extra/checkout_testdata.cmake]: indentation, it's now in an `if` loop

**Comment by doutriaux1** [comment]
@aashish24 we need this to test all pending branches with new baselines please approve quick.

**Pull request merged**